test/api-console.amf.test.js

```diff
@@ -71,6 +71,31 @@ describe('<api-console>', function() {
             assert.equal(result, 'List people');
           });
         });
+
+        describe('api-documentation', () => {
+          let element;
+          let amf;
+
+          before(async () => {
+            amf = await AmfLoader.load({ compact });
+          });
+
+          beforeEach(async () => {
+            element = await amfFixture(amf);
+          });
+
+          it('set rearrangeEndpoints to false', async () => {
```

Comment:
Could you also test how this work in a template? Add a fixture that has attributes set to test for correct data.